### PR TITLE
Configuration file watcher and JSON settings error improvement

### PR DESCRIPTION
**Pull request by nuest**

### CODE REVIEW

File: src/main/java/org/n52/iceland/config/json/JsonConfiguration.java [hunk 1]
```diff
@@ -251,4 +255,17 @@ private synchronized void persist() {
...
+
+    @Override
+    public Path getPath() {
+        return this.file.toPath();
+    }
+
+    @Override
+    public String toString() {
+        return MoreObjects.toStringHelper(this).omitNullValues()
+                .add("file", file).toString(); //To change body of generated methods, choose Tools | Templates.
```
Comment (autermann) [inline src/main/java/org/n52/iceland/config/json/JsonConfiguration.java]: Netbeans template...

File: pom.xml [hunk 2]
```diff
@@ -602,58 +596,58 @@
...
+                                    </pluginExecutionFilter>
+                                    <action>
+                                        <ignore></ignore>
+                                    </action>
+                                </pluginExecution>
+                            </pluginExecutions>
+                        </lifecycleMappingMetadata>
+                    </configuration>
+                </plugin>
+            </plugins>
```
Comment (autermann) [inline pom.xml]: Could you revert these? These are only space changes and will require a manual merge in #25...

File: pom.xml [hunk 3]
```diff
@@ -145,12 +145,6 @@
             <version>2.7</version>
         </dependency>
         <dependency>
-            <groupId>se.jiderhamn</groupId>
-            <artifactId>classloader-leak-prevention</artifactId>
-            <version>1.11.0</version>
-            <scope>runtime</scope>
-        </dependency>
-        <dependency>
```
Comment (autermann) [inline pom.xml]: Why did you remove this one?

Comment (nuest) [inline pom.xml]: There is no way to configure the classloader in iceland, it has to be done in an service implementation. The service implementation might also choose not to use the library.

Comment (autermann) [inline pom.xml]: Good point :+1:
